bbtools/38.86/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:xenial
+
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
```

Comment:
could you please adjust this line to `LABEL dockerfile.version="1"`

Reply:
I've added this

Reply:
ty!